### PR TITLE
HZN-705, HZN-1058: Upgrade to Camel 2.16, ActiveMQ 5.13

### PR DESCRIPTION
**Pull request by soleger**
Now that we're using Spring 4.1, this PR upgrades Camel and ActiveMQ to versions that roughly align with each other.

* JIRA: http://issues.opennms.org/browse/HZN-705
* JIRA: http://issues.opennms.org/browse/HZN-1058

### CODE REVIEW

File: core/daemon/src/main/resources/META-INF/opennms/applicationContext-daemon.xml [hunk 1]
```diff
@@ -29,13 +29,26 @@
   <onmsgi:reference id="eventProxy" interface="org.opennms.netmgt.events.api.EventProxy"/>
   <onmsgi:reference id="eventSubscriptionService" interface="org.opennms.netmgt.events.api.EventSubscriptionService"/>
 
+  <!-- TODO: Remove this bean after we stop using serialized object messages -->
```
Comment (j-white) [inline core/daemon/src/main/resources/META-INF/opennms/applicationContext-daemon.xml]: I thought we got rid of the remaining serialized object messages. Where do we still use them?

Comment (soleger) [inline core/daemon/src/main/resources/META-INF/opennms/applicationContext-daemon.xml]: I don't think that we do now but the original commits in this branch pre-dated that work. I'll track removing the ```org.apache.activemq.SERIALIZABLE_PACKAGES``` system properties in a new issue.

Comment (j-white) [inline core/daemon/src/main/resources/META-INF/opennms/applicationContext-daemon.xml]: Was this tracked anywhere?

Comment (soleger) [inline core/daemon/src/main/resources/META-INF/opennms/applicationContext-daemon.xml]: https://issues.opennms.org/browse/HZN-1071

**Pull request merged**